src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewResultExecutor.cs

```diff
@@ -176,6 +178,29 @@ public virtual Task ExecuteAsync(ActionContext actionContext, IView view, ViewRe
                 viewResult.StatusCode);
         }
 
+        /// <inheritdoc />
+        public async Task ExecuteAsync(ActionContext context, ViewResult result)
```

Comment:
Could we remove the now unused overload since the type was in `.Internal`?

Reply:
Sure